tests/gnrc_netif/main.c

```diff
@@ -1071,27 +1071,26 @@ static void test_netif_get_name(void)
     char exp_name[NETIF_NAMELENMAX + 1];
     char name[NETIF_NAMELENMAX];
     int res;
-    netif_t netif = netif_iter(NETIF_INVALID);
+    netif_t *netif = netif_iter(NULL);
     /* there must be at least one interface */
-    TEST_ASSERT(NETIF_INVALID != netif);
+    TEST_ASSERT(NULL != netif);
 
     res = netif_get_name(netif, name);
-    sprintf(exp_name, "if%d", (int)netif);
+    sprintf(exp_name, "if%d", (int) ((gnrc_netif_t*) netif)->pid);
     TEST_ASSERT_EQUAL_INT(strlen(exp_name), res);
     TEST_ASSERT_EQUAL_STRING(&exp_name[0], &name[0]);
-    TEST_ASSERT_EQUAL_INT(0, netif_get_name(INT16_MAX, name));
```

Comment:
This one is not required anymore, because `netif_t` pointers will always be valid or NULL